### PR TITLE
mod-status: show pstore logs on overview

### PR DESCRIPTION
**Pull request by selanf**

### CODE REVIEW

File: modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json [hunk 1]
```diff
@@ -96,6 +96,7 @@
 		"description": "Grant access to main status display",
 		"read": {
 			"file": {
+				"/sys/fs/pstore/*": [ "read" ],
```
Comment (feckert) [inline modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json:99]: Could you please move the acl into its on section `luci-mod-status-index-pstore`

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	}
+});
+
+return baseclass.extend({
+	title: _('Warning'),
+
+	load: function() {
+		return Promise.all([
+			L.resolveDefault(callFileList('/sys/fs/pstore/'), {}),
+			[]
```
Comment (feckert) [inline modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js:36]: Is this needed?

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		if (result[0].length<=0)
+			return;
+		var entries=result[0];
+		var content=result[1];
+
+		var filetabs = E('div', {}, E('div'));
+		for (var i=0; i<entries.length; i++) {
+			filetabs.firstElementChild.appendChild(E('div', { 'data-tab': entries[i], 'data-tab-title': entries[i] }, [
+				E('textarea', {
+					'id': 'pstore',
```
Comment (feckert) [inline modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js:56]: Should the id not be uniq

**Comment by selanf** [comment]
Screenshot with example files:
image

**Comment by selanf** [comment]
@feckert
> Is this needed?

In which way would you read the list?

> Should the id not be uniq

Not sure, i think within the div "data-tab" it is unique

> Could you please move the acl into its on section

Yes, the description of the section i used ""Grant access to main status display"" was confusing

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js (hunk 2 above)
Comment (selanf) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js:36]: In which way would you read the list?

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js (hunk 3 above)
Comment (selanf) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js:56]: Not sure, i think within the div "data-tab" it is unique

File: modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json (hunk 1 above)
Comment (selanf) [reply modules/luci-mod-status/root/usr/share/rpcd/acl.d/luci-mod-status.json:99]: Yes, the description of the section i used ""Grant access to main status display"" was confusing

**Comment by selanf** [comment]
notice to @dangowrt

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js (hunk 3 above)
Comment (selanf) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js:56]: ?

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js (hunk 2 above)
Comment (selanf) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js:36]: ?

**Pull request closed without merging**